locales/en.json

```diff
@@ -356,5 +356,11 @@
   "Enter the last two mailing addresses before your current address": "Enter the last two mailing addresses before your current address",
   "You must have used both addresses to file taxes. Choose a different question if you haven’t filed taxes from at least two other addresses.": "You must have used both addresses to file taxes. Choose a different question if you haven’t filed taxes from at least two other addresses.",
   "Mailing address 1": "Mailing address 1",
-  "Mailing address 2": "Mailing address 2"
-}
+  "Mailing address 2": "Mailing address 2",
+  "Details of your bankruptcy assignment": "Details of your bankruptcy assignment",
+  "Enter details about your bankruptcy assignment": "Enter details about your bankruptcy assignment",
+  "Name of your trustee in bankruptcy": "Name of your trustee in bankruptcy",
+  "Date of your bankruptcy assignment": "Date of your bankruptcy assignment",
+  "errors.login.trusteeLastName": "Please enter your trustee's last name",
+  "errors.login.trusteeFirst": "lease enter your trustee's first name"
```

Comment:
1. Can you move the error keys up above? I've been keeping them all near the top of the file (in `en` and `fr`).
2. Remember to switch the page to `fr` and reload.